config/default.yml

```diff
@@ -4701,6 +4701,7 @@ Style/RedundantParentheses:
   Description: "Checks for parentheses that seem not to serve any purpose."
   Enabled: true
   VersionAdded: '0.36'
+  VersionChanged: '<<next>>'
```

Comment:
This seems like a bugfix for the cop purposes, so it's unnecessary.

```suggestion
```

Reply:
The behavior of the cop is changing away from something that was hardcoded and tested (which will mean code that did not register an offense for 6 years will now). If you want me to still remove this I will but that was my rationale 😄

Reply:
I see, but it looks that `VersionChanged` is unnecessary as it is what end user wants it to be :-)

Reply:
Removed it.